src/parquet/column_reader.h

```diff
@@ -123,7 +124,7 @@ class TypedColumnReader : public ColumnReader {
   typedef typename type_traits<TYPE>::value_type T;
 
   TypedColumnReader(const parquet::ColumnMetaData* metadata,
-      const parquet::SchemaElement* schema, InputStream* stream) :
+      const parquet::SchemaElement* schema, std::unique_ptr<InputStream>& stream) :
```

Comment:
You're right. I wanted to avoid extra std::move() calls. This works now, but in the future we may want to pass a temporary to the ctor.